streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -266,7 +268,7 @@ void retryWithBackoff(final Map<TaskId, Set<TopicPartition>> tasksToBeCreated, f
                     } catch (final LockException e) {
                         // ignore and retry
                         if (!retryingTasks.contains(taskId)) {
-                            log.warn("{} Could not create task {}. Will retry: ", logPrefix, taskId, e);
+                            log.warn("{} Could not create task {} due to {}. Will retry: ", logPrefix, taskId, e);
```

Comment:
if you add the `{}` it will do `e.toString()` if you leave it out it will print the stack trace. So it depends on what we are trying to print here? In this case it is the `LockException` so it might be ok to just do `e.toString()`, rather than spamming the logs with the stack trace.